resources/views/checkout/partials/sidebar/totals.blade.php

```diff
@@ -1,26 +1,27 @@
 <div>
     <dt>@lang('Subtotal')</dt>
-    <dd>@{{ cart.prices.subtotal_including_tax.value | price }}</dd>
+    <dd v-cloak>@{{ cart.prices.subtotal_including_tax.value | price }}</dd>
```

Comment:
If we format the code like so we do not need a v-cloak, but can use a v-blur instead.
```suggestion
    <dd v-blur v-text="$options.filters.price(cart.prices.subtotal_including_tax.value )">&euro; 100,-</dd>
```
That way we blur a placeholder until it is replaced with the true value.

Then we won't have trouble with layout shifting for elements we know will exist anyways.

Reply:
This won't cause layout shifting, since it already has a title which is already rendered. And blurring Tax related is also tricky since it's configurable within magento. What do you think?